client/components/domains/use-your-domain-step/style.scss

```diff
@@ -15,6 +15,7 @@
 
 	.use-your-domain-step__option-inner-wrap {
 		display: flex;
+		flex-basis: 100%;
```

Comment:
Nice! I had a feeling not setting an initial value for this was causing the issue.